roles/htcondor/README.md

```diff
@@ -0,0 +1,23 @@
+### Deploy HTCondor
+
+### Requirements
+
+- Ansible
+
+### Recommended Configuration for the HTCondor machine
+
+- OS: CentOS 7 or Alma 9
+
+### Configuration Specifications
+
+- Token and GSI authentication flags both default to False unless otherwise specified
+- Configuration without GSI authentication will install HTCondor Version 23.*
+- Configuration with GSI authentication will install HTCondor Version 8.*
+- GSI authentication is not available for Alma 9 due to its incompatability with HTCondor Version 8.*
+
+### Steps to install csv2 on a pre-configured host
```

Comment:
Why is csv2 needed for a HTCondor install?

Reply:
Should be fixed now